frontend/compositions/profile-content/profile-info.tsx

```diff
@@ -0,0 +1,80 @@
+import * as React from "react"
+import styles from "./profile-content.module.css"
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome"
+import { faEdit } from "@fortawesome/free-solid-svg-icons"
+import { UserProfileProps, emptyUser } from "../../models/profile"
+
+export default function ProfileInfo({ userData = emptyUser }: UserProfileProps) {
+  const [editMode, setEditMode] = React.useState(false)
+  const { profileData, sectionTitle, row } = styles
+  const { firstName, lastName, email, phone } = userData
+
+  function handleSubmit(e: React.FormEvent) {
+    // TODO: update user
+  }
+
+  if (editMode) {
+    return (
+      <div className={profileData}>
+        <header className={sectionTitle}>Edit Your Account Information</header>
+        <form onSubmit={handleSubmit}>
```

Comment:
lets remove the onSubmit on the form, form has sideeffects when doing a POST request, place the `handleSubmit` on a normal button instead please